BurnOutSharp/PackerType/WinZipSFX.cs

```diff
@@ -47,7 +51,7 @@ public string CheckContents(string file, byte[] fileContent, bool includePositio
             return null;
         }
 
-        private static string GetV3PlusVersion(byte[] fileContent, int position)
+        private static string GetV3PlusVersion(byte[] fileContent, int position, int position2)
```

Comment:
You don't use `int position` in this method ever, so you can remove it as a parameter. Also, the naming should be a little more consistent with what `position2` is, in this case. For example, it's `xmlStartPosition` in this case.